pandas/tests/io/json/test_normalize.py

```diff
@@ -526,8 +526,8 @@ def test_non_list_record_path_errors(self, value):
         test_input = {"state": "Texas", "info": parsed_value}
         test_path = "info"
         msg = (
-            f"{test_input} has non list value {parsed_value} for path {test_path}. "
-            "Must be list or null."
+            f"Path must contain list or null, "
+            f"but got {type(test_input['info']).__name__} at {repr(test_path)}"
```

Comment:
Could you fully evaluate `{type(test_input['info']).__name__}` and `{repr(test_path)}` here?

Reply:
Sure. BTW, I am not sure if the failed test is introduced by me or not. Should I fix it?

Reply:
I just changed `{type(test_input['info']).__name__}` to `{type(parsed_value).__name__}` since there are several tests based on different parameters and I can't fix the type name